roles/monitoring_client/tasks/install_node_exporter.yml

```diff
@@ -45,47 +45,38 @@
     state: started
     enabled: true
   when: ansible_service_mgr == "systemd"
-
-- name: Check 'passlib' is installed on the Ansible controller
-  delegate_to: localhost
-  run_once: true
-  changed_when: false
-  ansible.builtin.shell: |
-    set -o pipefail
-    {{ ansible_playbook_python }} -m pip freeze | grep passlib
-  register: passlib_exists
-  # grep returns error code 2 for exceptions, otherwise don't want failure
-  # https://stackoverflow.com/a/41010653/7359333
-  failed_when: "passlib_exists.rc == 2"
-
-- name: Fail if 'passlib' is missing on the Ansible controller
-  run_once: true
-  ansible.builtin.fail:
-    msg: >
-      'bcrypt' algorithm requires 'passlib' on the Ansible controller,
-      run `python -m pip install passlib` and re-run the script
-  when: "passlib_exists.rc == 1"
+  tags:
+    - molecule-idempotence-notest
```

Comment:
does this task fail idempotency checks because the daemon is always reloaded?

Reply:
I think that must be it - Ansible always flags a change so the `idempotence` test fails.